inventory/decorators.py

```diff
@@ -0,0 +1,15 @@
+from django.core.exceptions import PermissionDenied
+from inventory.models import *
+
+
+def user_is_item_creator(function):
```

Comment:
Georges's decorator would only be effective to prevent someone who is not logged in to edit an item; however, if he is already logged in he will be able to edit all items even the items he did not add. That's why I wrote a new decorator to prevent a logged-in user from editing items added by other users.